packages/ui/Slottable/useSlots.js

```diff
@@ -82,7 +83,7 @@ function distribute (slots, {children, ...fallback}) {
 	// will append to existing props and we want the distributed value to override the fallback
 	// value.
 	return {
-		...fallback,
+		...slots,
```

Comment:
What about deleting `slots` props whose value is `undefined`?

Reply:
Yeah, that's probably necessary.